cmd/minikube/cmd/start.go

```diff
@@ -413,7 +413,7 @@ func showKubectlInfo(kcs *kubeconfig.Settings, k8sVersion string, machineName st
 
 	path, err := exec.LookPath("kubectl")
 	if err != nil {
-		out.T(style.Tip, "kubectl not found. If you need it, try: 'minikube kubectl -- get pods -A'")
+		out.T(style.Tip, "kubectl not found. If you need it, try: 'minikube kubectl --'")
```

Comment:
Personally, given the strange syntax, I don't feel like this is obvious enough. I wanted to give an example that a user is likely to understand without consulting additional documentation.

Reply:
Maybe 'minikube kubectl -- help'?

Reply:
Beauty, as they say, is in the eye of the beholder. Maybe I should have used a different word (than ugly).

Did you know that `kubectl`, `kubectl --help` and `kubectl help` all have the _same_ output, by the way ?

I like "help" as an improvement (over "get"), but it wasn't really what I meant.

Maybe I will just close this PR, it was just the new screenshot that caused it.

![screenshot](https://raw.githubusercontent.com/kubernetes/minikube/8dadb5984354b4cce482ba4d11202d9fff7fc783/site/static/images/screenshot.png)

It looked a bit like an error, while it is perfectly normal. But maybe we want to avoid "other solutions" like:

```

Command 'kubectl' not found, but can be installed with:

sudo snap install kubectl

```

It just seemed weird to give an example command, rather than show how to "install" it (e.g. as an alias).